fri/benches/fold_even_odd.rs

```diff
@@ -1,35 +1,38 @@
-use core::any::type_name;
+use std::marker::PhantomData;
```

Comment:
Nit: I know expecting benches to be `no_std` is unreasonable, still at the moment `grep std::` returns basically nothing and it would be nice to keep it that way.
```suggestion
use core::marker::PhantomData;
```